node/src/lib.rs

```diff
@@ -29,6 +29,7 @@ mod listener_handler;
 pub mod masquerader;
 pub mod neighborhood;
 pub mod node_configurator;
+#[cfg(test)]
```

Comment:
I'd just check out if that file itself doesn't have `#![cfg(test)]` in it at the top, then I might pick just one to stay.

Reply:
Good eye.